specs/index.md

```
@@ -0,0 +1,71 @@
+新規仕様書作成時は以下のテンプレートを使用:
+
+- [.specify/templates/spec.md.tpl](../.specify/templates/spec.md.tpl)
+- [.specify/templates/plan.md.tpl](../.specify/templates/plan.md.tpl)
+- [.specify/templates/tasks.md.tpl](../.specify/templates/tasks.md.tpl)
+
+## 関連ドキュメント
+
+- [CLAUDE.md](../CLAUDE.md) - プロジェクト概要
+- [constitution.md](../.specify/memory/constitution.md) - 品質・非機能要件
+- [architecture.md](../.specify/memory/architecture.md) - アーキテクチャ設計
+- [progress.md](../docs/progress.md) - 進捗ログ
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)